node.go

```diff
@@ -57,6 +57,11 @@ type Config struct {
 	// Optional sharder to synchronize cluster changes to. Synchronization of the
 	// Sharder happens prior to Observers being notified of changes.
 	Sharder shard.Sharder
+
+	// Optional identifier to prevent clusters from accidentally merging.
+	// Nodes are prevented from joining a cluster with an explicit label if
+	// they do not share the same label.
+	Label string
 }
```

Comment:
Should this change be in this PR?

Reply:
thanks for pointing this out, that was a merging error, should be good now